aea/cli_gui/__init__.py

```diff
@@ -110,8 +111,8 @@ def _sync_extract_items_from_tty(pid: subprocess.Popen):
     output = []
     err = ""
     for line in io.TextIOWrapper(pid.stdout, encoding="utf-8"):
-        if line[:6] == "Name: ":
-            item_ids.append(line[6:-1])
+        if line[:11] == "Public ID: ":
```

Comment:
the CLI GUI was using only the package names, hence was needed to fix the parsing from `aea list` to take the public ID instead of the name.